### PR TITLE
[DebuggerV2] Add inactive component

### PR DESCRIPTION
**Pull request by caisq**
* Motivation for features / changes
  * Develop DebuggerV2 plugin

* Technical description of changes
  * Add a "stateless" (static) component according to the inactive screen in the mockups by @baileydesign
  * The screenshots in the three "exhibits" in the middle are omitted for now. They'll be added when we have the actual FE components working.

* Screenshots of UI changes
image

### CODE REVIEW

**Comment by caisq** [comment]
cc @baileydesign

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+==============================================================================*/
+
+.arg {
+  color: lightblue;
+  font-style: italic;
+  margin: 2px;
+}
+
+.code {
+  font-family: Roboto Mono;
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: Put quote around and specify a fallback font (monospace). 

> The name of a font family. For example, "Times" and "Helvetica" are font families. Font family names containing whitespace should be quoted.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-family

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  font-size: 15px;
+  padding: 50px;
+}
+
+.details-container {
+  vertical-align: middle;
+}
+
+.details {
+  align-content: left;
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: `align-content` does not do anything on non-flexbox and non-grid context. Extraneous?

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  display: inline-block;
+  margin: 30px;
+  max-width: 360px;
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: is both width and max-width required?

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  width: 100%;
+}
+
+.exhibit {
+  align-content: center;
+  display: inline-block;
+  margin-top: 10px;
+  margin-left: 60px;
+  margin-right: 60px;
+  margin-bottom: 10px;
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: `margin: 10px 60px;`?

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  width: 220px;
+}
+
+.exhibit .description {
+  font-weight: bold;
+  text-align: center;
+  width: 220px;
+}
+
+.exhbit .screenshot canvas {
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: typo. `exhbit` -> `exhibit`. Is this rule actually doing anything?

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  height: 210px;
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: Prefer to set the dimension relative to the container and minimize number of hard coded width/height.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css (hunk 1 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: Done.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css (hunk 4 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: Done.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css (hunk 5 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: Fixed it. Good catch.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css (hunk 6 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: In this case I do need to specify an explicit height. Width has been made relative, though.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css (hunk 3 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: Removed.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css (hunk 2 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css]: Removed.

**Pull request merged**